src/par_iter/util.rs

```diff
@@ -1,25 +0,0 @@
-use std::marker::PhantomData;
-
-/// A little structure used to mark some type. This is often used when
-/// we have some phantom type we wish to remember as part of a struct.
-/// This doesn't represent "reachable data", and hence it is
-/// considered sendable regardless of `T`, unlike `PhantomData`.
-pub struct PhantomType<T> {
-    data: PhantomData<T>
-}
```

Comment:
I'm really glad to see the end of this. Seemed like a dangerous subversion of the type system.